mlflow/metrics/genai/prompts/v1.py

```diff
@@ -17,33 +17,38 @@
 }
 grading_system_prompt_template = PromptTemplate(
     """
-Please act as an impartial judge and evaluate the quality of the provided output which
-attempts to produce output for the provided input based on a provided information.
+Task
```

Comment:
Is there strong evidence for removing colons / underlines here? A colon or underline seems much more natural than nothing

Reply:
colons would be helpful, underline would cost more tokens.